ofxdump/Makefile.am

```diff
@@ -41,15 +41,19 @@ TESTS = \
  ../doc/ofx_sample_files/ofx-2.1.1-sect-16.5.6.5.ofx \
  ../doc/ofx_sample_files/ofx-2.1.1-sect-8.5.5.ofx \
  ../doc/ofx_sample_files/failing-ofx-file1.xml \
- ../doc/ofx_sample_files/failing-ofx-file2.xml
+ ../doc/ofx_sample_files/failing-ofx-file2.xml \
+ ../doc/ofx_sample_files/null-ptr-deref-1.ofx \
+ ../doc/ofx_sample_files/null-ptr-deref-2.ofx
 
```

Comment:
Very good!

One final addition that would be nice: Could you also add those file names to ofxdump/CMakeLists.txt into the variable SAMPLE_OFX_FILENAMES_FAILING ? Then not only the autotools testsuite runs those, but also the CMake test suite. Thanks!